### ISSUE TITLE
Ui integration for support Trezor, Ledger, Keepkey and status-keycard

### ISSUE THREAD

**Issue by danimesq** [bisq-network/bisq#3360]

**Comment by danimesq** [comment]
@freimair

**Comment by chriscrutt** [comment]
please

**Comment by MarekPasnikowski** [comment]
This is an amusing discussion to an outsider like me.

Anyway, I own a Trezor and I refuse to use any software without integrated support for hardware wallets. The wiki has only one mention of a hardware wallet use.
> Do not send BSQ to hardware wallets.

Is there any chance to encourage a prioritized integration of Trezor in Bisq? One idea I have is I could buy a significant amount of BSQ to fund the feature development.

**Comment by ripcurlx** [comment]
> This is an amusing discussion to an outsider like me.
> 
> Anyway, I own a Trezor and I refuse to use any software without integrated support for hardware wallets. The wiki has only one mention of a hardware wallet use.
> 
> > Do not send BSQ to hardware wallets.
> 
> Is there any chance to encourage a prioritized integration of Trezor in Bisq? One idea I have is I could buy a significant amount of BSQ to fund the feature development.

Thanks for the offer to fund this development, but it is more suited for Bisq2, which is already under development: https://github.com/bisq-network/bisq2

**Comment by MarekPasnikowski** [comment]
I had no idea about the version 2. I will take a look at it.

**Comment by MarekPasnikowski** [comment]
I have given this issue some time to think about it. I can see two distinct problems to solve:

 1. Implementation of BSQ tokens in the hardware wallets. [This is the Trezor's Wiki page about new coins.](https://wiki.trezor.io/Developers_guide:Contributing)

    > This is what I am most interested in. I could tolerate keeping the small amounts of cryptocurrency I would want to trade outside of Trezor. However, I will not compromise the security of the long term holdings, which would include BSQ.

 2. The actual integration of hardware wallets in the BISQ application. This implies exclusive usage of hardware keys in all transactions. Definitely a big project and makes a lot of sense to do it in the BISQ2 version.

In conclusion, my main question is:

> How to approach the implementation of BSQ in Trezor, and other wallets?

 * Should a random person with interest (like me) ask the wallet manufacturer to add the support?
 * Should a BISQ representative contact the manufacturer?
 * Should it be left to a developer to decide to implement it at an undefined time?
